### PR TITLE
Intro

### PR DESCRIPTION
**Pull request by perrygreenfield**
initial draft versions of introduction and final section

### CODE REVIEW

**Comment by WilliamJamieson** [comment]
Can you rebase this?

File: notebooks/02-Introduction.ipynb [hunk 1]
```diff
@@ -0,0 +1,193 @@
+{
```
Comment (WilliamJamieson) [inline notebooks/02-Introduction.ipynb:1]: <p>Add periods?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (WilliamJamieson) [inline notebooks/02-Introduction.ipynb:1]: <p>Again add periods or remove them?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (WilliamJamieson) [inline notebooks/02-Introduction.ipynb:1]: <p>Block index should be mentioned.</p><p> </p><p>Also is there any way to show a "diagram" of the low-level file layout, kind of how we now display the files in the asdf documentation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (WilliamJamieson) [inline notebooks/02-Introduction.ipynb:1]: <p>Mention how this is currently supported using the <code>astropy.modeling</code> package </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

File: notebooks/07-Advanced_Topics_and_Plans.ipynb [hunk 2]
```diff
@@ -0,0 +1,296 @@
+{
```
Comment (WilliamJamieson) [inline notebooks/07-Advanced_Topics_and_Plans.ipynb:1]: <p>Divide up and use markdown title?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (WilliamJamieson) [inline notebooks/07-Advanced_Topics_and_Plans.ipynb:1]: <p>Add markdown title</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

**Comment by zacharyburnett** [comment]
should this be based on @WilliamJamieson's Chapter 4 branch? if so then we should merge that branch first and rebase this onto the new `main` afterwards, to prevent duplicate commits

**Comment by perrygreenfield** [comment]
It was rebased against main. Should I have done something differnently?

**Comment by WilliamJamieson** [comment]
Thats correct. I think it is an artifact of the accidental merger of my PR the other day.

**Comment by zacharyburnett** [comment]
oh you are correct, I apologize, it looks like the Files Changed doesn't show any of the other changes.

File: notebooks/02-Introduction.ipynb (hunk 1 above)
Comment (perrygreenfield) [reply notebooks/02-Introduction.ipynb:1]: <p>done.</p>

Comment (perrygreenfield) [reply notebooks/02-Introduction.ipynb:1]: <p>done</p>

Comment (perrygreenfield) [reply notebooks/02-Introduction.ipynb:1]: <p>done I think.</p>

File: notebooks/07-Advanced_Topics_and_Plans.ipynb (hunk 2 above)
Comment (perrygreenfield) [reply notebooks/07-Advanced_Topics_and_Plans.ipynb:1]: <p>Check to see if I did what you intended.</p>

Comment (perrygreenfield) [reply notebooks/07-Advanced_Topics_and_Plans.ipynb:1]: <p>likewise here.</p>

File: notebooks/02-Introduction.ipynb [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
```
Comment (nden) [inline notebooks/02-Introduction.ipynb:1]: <p><code>remants of punch&nbsp;</code> --&gt; remnants of punch&nbsp; </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (nden) [inline notebooks/02-Introduction.ipynb:1]: <p>The indentation in the list does not come out correct.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

**Comment by nden** [comment]
These should move out of `notebooks` into their own directories. It'd be good if you can split the two notebooks into two PRs. I think `intro` is almost ready while I expext ore material to be added to `Advanced`. They may be ready to merge at different times.

File: notebooks/02-Introduction.ipynb (hunk 3 above)
Comment (perrygreenfield) [reply notebooks/02-Introduction.ipynb:1]: <p>Could you clarify this?</p>

**Pull request closed without merging**